src/_base/helm/app/templates/application/webapp/ingress.yaml

```diff
@@ -1,27 +1,29 @@
 {{ if and (index .Values.services .Values.ingress.target_service "enabled") (eq .Values.ingress.type "standard") }}
+{{- with .Values.services.nginx -}}
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-{{- if .Values.services.nginx.ingress.annotations }}
+{{- if .ingress.annotations }}
   annotations:
-  {{- range $key, $value := .Values.services.nginx.ingress.annotations }}
+  {{- range $key, $value := .ingress.annotations }}
     {{ $key }}: {{ $value | quote }}
   {{- end }}
 {{- end }}
   creationTimestamp: null
   labels:
     {{- include "chart.labels" $ | nindent 4 }}
     app.kubernetes.io/component: webapp
-    app.service: {{ .Values.resourcePrefix }}webapp
-  name: {{ .Values.resourcePrefix }}webapp
+    app.service: {{ $.Values.resourcePrefix }}webapp
```

Comment:
what does `$` do different here?

Reply:
We've added a with on line two, which scopes `.` to be `.Values.services.nginx.*`

The `$` refers to the root context (like we had before `with`)

Reply:
ah nice, pardon my poor twig knowledge 😄

Reply:
This one is just helm/gotpl :)